### PR TITLE
edit read

### PR DESCRIPTION
**Pull request by TakefumiYamamura**
Review Me @TsubasaK111 
I added the explanation of environmental variables.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,4 +169,20 @@ codecheckでは、あなたが工夫して解答した点を[answer.md](answer.m
...
+  キーを実装物にハードコードしたままgithubにpushしてしまうと、  
+  個人用のAPIキーなどの情報が公開されてしまい、悪用される可能性が発生します。
+- 代わりに、codecheckの環境変数機能をご利用ください。  
+  秘匿したい情報をgithub上に公開せずにcodecheck内で利用し、  
+  正しく動作するコードをcodecheck上に提出することが可能になります。
+- 試験のチャレンジ一覧ページ、チャレンジ横にドロップメニューのボタンが表示されます。
+  ![exam_challenge_list_masked_sm](https://user-images.githubusercontent.com/7766684/31326767-4145f402-ad05-11e7-8f38-fc55913a2776.png)
+- そのボタンをクリックすると、「環境変数の設定」という項目が表示されます。  
+ ![triangle_dropdown](https://user-images.githubusercontent.com/7766684/31326777-49570ec4-ad05-11e7-914c-d61b245109dd.png)
+- 「環境変数の設定」をクリックし、任意の命名でAPIキーを環境変数として設定ください。
```
Comment (TsubasaK111) [inline README.md]: `APIキーを環境変数として` => `APIキーなどの環境変数を`

**Review by TsubasaK111 — APPROVED** [review]
@takayukioda review me!

**Review by takayukioda — APPROVED** [review]
LGTM

**Comment by takayukioda** [comment]
I would like to have "index" though

**Pull request merged**